src/networking.c

```diff
@@ -32,10 +32,16 @@
 #include "cluster.h"
 #include "script.h"
 #include "fpconv_dtoa.h"
+#include <net/if.h>
 #include <sys/socket.h>
 #include <sys/uio.h>
 #include <math.h>
 #include <ctype.h>
+#include <ifaddrs.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+
+#define NETWORK_LOOPBACK_MASK (IFF_RUNNING|IFF_LOOPBACK|IFF_UP)
```

Comment:
:star_struck:

nit: you could place the `define` next to the related function, as is done for `PROTO_DUMP_LEN` for example.